source/mongoshrc.txt

```diff
@@ -18,35 +18,6 @@
 
 .. include:: /includes/fact-mongoshrc-loading.rst
 
-Examples
---------
-
-The following examples create custom prompts. To display:
-
-- :ref:`line numbers <ex-display-line-numbers>`
-- :ref:`database and hostname <ex-display-dbname-and-hostname>`
-- :ref:`system uptime and document count <ex-display-system-up-time>`
-
-add the example code to :ref:`.mongoshrc.js <mongoshrc-js>`.
-
-.. _ex-display-line-numbers:
-
-Display Line Numbers
-~~~~~~~~~~~~~~~~~~~~
-
-.. include:: /includes/examples/ex-display-line-numbers.rst
-
-.. _ex-display-dbname-and-hostname:
-
-Display Database and Hostname
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. include:: /includes/examples/ex-display-dbname-and-hostname.rst
-
-.. _ex-display-system-up-time:
-
-Display System Up Time and Document Count
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-.. include:: /includes/examples/ex-display-system-uptime.rst
-
+You can use the ``.mongoshrc`` file to customize ``mongosh`` startup
```

Comment:
I don't see the ref where these examples are linked. Could it have been dropped by mistake?

Reply:
I intentionally removed the examples from this page to convert it to strictly a reference page specific to the .mongoshrc file. The examples are covered in these two places:

- https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-23224/write-scripts/#std-label-mongosh-write-scripts-config-file
- https://docs-mongodbcom-staging.corp.mongodb.com/mongodb-shell/docsworker-xlarge/DOCSP-23224/reference/customize-prompt/

There is a link to the Write Scripts section on the reference page. I'll add a heading so it's a bit more obvious.